docs/source/index.md

```diff
@@ -1,21 +1,99 @@
-# Welcome to MALA's documentation!
+![image](./img/logos/mala_horizontal.png)
+
+# Welcome to MALA!
+
+## What is MALA?
+
+MALA is a software package for building ML models that replace
+density functional theory (DFT) calculations. DFT is **the** most
+popular method for simulating materials on a quantum level and predicting 
+materials properties, and employed by researcher throughout the world. 
+
+While DFT is very efficient compared to other quantum mechanical simulation
+techniques, it still scales cubically with number of atoms and temperature 
+simulated, making large scale simulation at arbitrary temperatures very 
+computationally challenging. 
+
+Therefore, a new direction of research tries to [combine the power of ML with DFT](https://journals.aps.org/prmaterials/abstract/10.1103/PhysRevMaterials.6.040301).
+MALA is at the forefront of these efforts, being the first library to allow 
+users to build ML-DFT models that give **full access to the electronic structure**,
+observables such as the total free energy of the system **and** scale well with
+system size and temperature.
+
+## How does MALA work? 
+
+MALA operates by predicting the so called local density of states (LDOS). 
+The LDOS is an efficient and versatile representation of the electronic 
+structure of a material. Based on the LDOS, a range of useful observables can
+be calculated. 
+
+MALA calculates a grid-based representation of any atomic structure presented 
+to it (so called descriptors). These descriptors are then passed through a 
+neural network, a very efficient ML algorithm, to predict the LDOS. This is 
+done individually at each point of the real space grid, allowing for easy 
+scalability and parallelization. 
+
+Afterwards, a range of post-processing features are available within MALA to
+calculate obervables from interest. Just like a DFT calculation - just faster.
+For more detail on the workflow, check out the [first MALA publication](https://www.doi.org/10.1103/PhysRevB.104.035120).
+
```

Comment:
Subsequently, MALA offers a variety of post-processing features to compute desired observables. Similar to a DFT calculation, but with enhanced speed. For further details on the workflow, refer to the [first MALA publication](https://www.doi.org/10.1103/PhysRevB.104.035120).